### ISSUE TITLE
RLS is not being detected

### ISSUE THREAD

**Issue by adamski234** [rust-lang/rls-vscode#761]
I installed rustup, the extension and opened a new project (`cargo init`). I got the `RLS is not installed` notification and accepted the prompt for installation. Terminal opened, installed `rust-analysis`, and nothing happened. The spinner was still spinning, stuck at installing. After a restart, I got prompted to install rls again, this time with the terminal output saying it was installed. 
Reinstallation didn't work.
Dirty fix that seems to work for now:
I manually installed `rls rust-analysis rust-src` using rustup. ~~Then I edited the [`$HOME/.vscode-oss/extensions/rust-lang.rust-0.7.0/out/src/rustup.js`](https://github.com/rust-lang/rls-vscode/blob/master/src/rustup.ts#L145) file and replaced the linked line with `return true`.~~ This was not necessary for it to work. RLS seems to be started and work properly.

System info:
[code](https://www.archlinux.org/packages/community/x86_64/code/) version: 1.44.0-1
Extension version: 0.7.0

**Comment by demosdemon** [comment]
exact same scenario for me

```
Version: 1.44.0
Commit: 2aae1f26c72891c399f860409176fe435a154b13
Date: 2020-04-08T08:23:56.137Z
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.4.0

rustup 1.21.1 (2020-01-08)

Default host: x86_64-apple-darwin
rustup home:  /Users/brandon/.rustup

stable-x86_64-apple-darwin (default)
rustc 1.42.0 (b8cedc004 2020-03-09)

Name: Rust (rls)
Id: rust-lang.rust
Description: Rust language support - code completion, Intellisense, refactoring, reformatting, errors, snippets. A client for the Rust Language Server, built by the RLS team.
Version: 0.7.0
Publisher: rust-lang
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=rust-lang.rust
```

**Comment by demosdemon** [comment]
oh, this is a dupe of #760

**Comment by Xanewok** [comment]
Fixed in https://github.com/rust-lang/rls-vscode/commit/800a04afd2629146ccc18013ab36f978705d20cf, released a 0.7.2 with the fix. It seems that this has regressed in the newest VSCode.
